views/jira-manual-app-creation.hbs

```diff
@@ -0,0 +1,142 @@
+          <label for="client-id">GitHub client ID</label>
+          <input class="text full-width-field" type="text" required id="client-id" name="client-id" />
+        </div>
+
+        <div class="jiraManualAppCreation__formTitle">URLs</div>
+        <div class="jiraManualAppCreation__formSubTitle">
+          Copy and paste the following URLs into your GitHub Enterprise Server configuration. This is unique to your site.
+        </div>
+
+        <div class="field-group">
+          <label for="callback-url">Callback URL</label>
+          <!--TODO: replace the value by actual data-->
+          <input class="text full-width-field" type="text" id="callback-url"
+                 name="callback-url" value="https://github.atlassian.com/github/callback/V1StGXR8_Z5jdHi6B-myT"
+                 readonly
+          />
+        </div>
+
+        <div class="field-group">
+          <label for="setup-url">Setup URL</label>
+          <!--TODO: replace the value by actual data-->
+          <input class="text full-width-field" type="text" id="setup-url"
+                 name="setup-url" value="https://github.atlassian.com/github/setup/V1StGXR8_Z5jdHi6B-myT"
+                 readonly
+          />
+        </div>
+
+        <div class="field-group">
+          <label for="webhook-url">Webhook URL</label>
+          <!--TODO: replace the value by actual data-->
+          <input class="text full-width-field" type="text" id="webhook-url"
+                 name="webhook-url" value="https://github.atlassian.com/github/webhook/V1StGXR8_Z5jdHi6B-myT"
+                 readonly
+          />
+        </div>
+
+        <div class="jiraManualAppCreation__formTitle">Secrets</div>
+        <div class="jiraManualAppCreation__formSubTitle">
+          Paste secrets from your GitHub Enterprise Server configuration.
+        </div>
+
+        <div class="field-group">
+          <label for="client-secret">GitHub client secret</label>
+          <input class="text full-width-field" type="password" required id="client-secret" name="client-secret" />
+        </div>
+
+        <div class="field-group">
+          <label for="webhook-secret">Webhook secret</label>
+          <input class="text full-width-field" type="password" required id="webhook-secret" name="webhook-secret" />
+        </div>
+
+        <div class="field-group">
+          <label>Private key</label>
+          <div class="jiraManualAppCreation__formFileDropArea">
+            <img class="jiraManualAppCreation__formFileIcon" src="/public/assets/file-upload.svg" alt="Upload">
+            <div>
+              <div class="jiraManualAppCreation__formFileMsg">Upload your private key</div>
+              <div class="jiraManualAppCreation__formFileSubMsg">Upload your generated private key from GitHub</div>
+            </div>
+            <div class="aui-button">Upload</div>
+            <input class="jiraManualAppCreation__formFileInput" id="private-key" required type="file">
```

Comment:
Can we get a screenshot/recording of what this looks like when a file has been uploaded?

Reply:
Video has been added in the description